pkg/genevalogging/genevalogging.go

```diff
@@ -28,10 +28,13 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/tls"
 )
 
+// note: any changes to MDSD or Fluentbit config should also be applied in openshift/installer config or unit files.
+
 const (
 	kubeNamespace      = "openshift-azure-logging"
 	kubeServiceAccount = "system:serviceaccount:" + kubeNamespace + ":geneva"
 
+	//when replacing these two image formats also replace the one in /pkg/backend/openshiftcluster/create.go getBootstrapLoggingConfig()
```

Comment:
Instead of using a comment to enforce this, can we import these values in both places from a separate package?

Reply:
+1.  Please put this in pkg/util/version/const.go.
Please can you also move InstallStream and Streams values from pkg/util/version/stream.go to that file as well?

Reply:
moved to /pkg/util/version/const.go